plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py

```diff
@@ -67,12 +93,21 @@ def uiCreate(cls, parentLayout, relativeToWhat):
         kFileOptionsStr = getMayaUsdString("kFileOptions")
         kMakePathRelativeStr = getMayaUsdString("kMakePathRelativeTo" + relativeToWhat)
         kMakePathRelativeAnnStr = getMayaUsdString("kMakePathRelativeTo" + relativeToWhat + "Ann")
+        kUnresolvedPathStr = getMayaUsdString("kUnresolvedPath")
+        kUnresolvedPathAnnStr = getMayaUsdString("kUnresolvedPathAnn")
+        kResolvedPathStr = getMayaUsdString("kResolvedPath")
+        kResolvedPathAnnStr = getMayaUsdString("kResolvedPathAnn")
  
         optBoxMarginWidth = mel.eval('global int $gOptionBoxTemplateDescriptionMarginWidth; $gOptionBoxTemplateDescriptionMarginWidth += 0')
         cmds.setParent(topForm)
         cmds.frameLayout(label=kFileOptionsStr, collapsable=False)
         widgetColumn = cmds.columnLayout()
-        cmds.checkBox(cls.kMakePathRelativeCheckBox + relativeToWhat, label=kMakePathRelativeStr, ann=kMakePathRelativeAnnStr)
+        cmds.checkBox(cls.kMakePathRelativeCheckBox, label=kMakePathRelativeStr, ann=kMakePathRelativeAnnStr)
+        if relativeToWhat == 'SceneFile':
+            cmds.checkBox(cls.kMakePathRelativeCheckBox, edit=True, changeCommand=cls.onMakePathRelativeChanged)
+            cmds.textFieldGrp(cls.kUnresolvedPathTextField, label=kUnresolvedPathStr, ann=kUnresolvedPathAnnStr, editable=False)
+            cmds.textFieldGrp(cls.kResolvedPathTextField, label=kResolvedPathStr, ann=kResolvedPathAnnStr , editable=False)
+            cls._haveRelativePathFields = True
```

Comment:
For now only add the preview fields for the SceneFile. Design is still working on the preview for the edit layer target case.